spec/policies/idv/gpo_verify_by_mail_policy_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe Idv::GpoVerifyByMailPolicy do
+  describe '#gpo_available_for_user?' do
+    let(:subject) { described_class.new(user).gpo_available_for_user? }
+    let(:user) { create(:user) }
+
+    context 'when the feature flag is off' do
+      before do
+        allow(IdentityConfig.store).to receive(:enable_usps_verification)
+          .and_return false
+      end
+
+      it 'returns false' do
+        expect(subject).to eq false
+      end
+    end
+
+    context 'when the feature flag is on' do
+      before do
+        allow(IdentityConfig.store).to receive(:enable_usps_verification)
+          .and_return true
+      end
+
+      context 'when the user is rate limited' do
+        # copypasta
+        before do
+          enqueue_gpo_letter_for(user, at_time: 4.days.ago)
+          enqueue_gpo_letter_for(user, at_time: 3.days.ago)
+          enqueue_gpo_letter_for(user, at_time: 2.days.ago)
+        end
+
+        it 'returns false' do
+          expect(subject).to eq false
+        end
+      end
+
+      context 'when the user has a too-old profile' do
+        before do
+          create(:profile, :verify_by_mail_pending, user: user, created_at: 90.days.ago)
+        end
```

Comment:
The norm seems to avoid mocking when we can. But I wonder if, for now, it would actually be more proper to do so, since we're relying on the GpoMail class which already has a proper test for these.